README.md

```diff
@@ -188,6 +192,12 @@ Fatigue is returned in the range of 0 - 1; therefore a fatigue value of 1 and co
 Higher makes fatigue build up far more fog, lower makes fatigue matter less in terms of fogging.\
 default: 1
 
+### cbrn_kat_enabled
+
+Boolean, the config will detect wheather KAT - Advanced Medical is installed or not.
+True will cause the script to read and use KAT chemical warfare data and injuries when neccessary.
+default: auto
```

Comment:
Wanna add your self to the contributor list at the bottom?

Reply:
I'll do so later. I have another update for medication to reduce cbrn_damange as well. I thought putting it together with this pull request would make it too messy.